ts/util/accountManager.ts

```diff
@@ -164,6 +164,10 @@ async function createAccount(identityKeyPair: any) {
   // Enable typing indicators by default
   await Storage.put(SettingsKey.settingsTypingIndicator, false);
 
+  // opengroups pruning in ON by default on new accounts, but can change that from the settings
```

Comment:
Small typos should be "opengroups pruning is ON by default on new accounts, but you can change that from the settings"